substrate/frame/sassafras/src/data/tickets-sort.md

```diff
@@ -267,7 +267,7 @@ buffer (i.e. how many tickets we retain from the last processed segment)
 | 3 | 14 |
 | 2 | 17 |
 | 1 | 9 |
-| 0 | 13
+| 0 | 13 |
```

Comment:
This change doesn't seem to be part of the commits of 2042 PR https://github.com/paritytech/polkadot-sdk/pull/2942/files